lib/util.js

```diff
@@ -240,3 +240,7 @@ util.removeTrailingSlash = function(path) {
   }
   return path;
 };
+
+util.isIE11OrLess = function(path) {
+  return document.documentMode && document.documentMode <= 11;
```

Comment:
since we don't support ie8/9 anyway, I don't bother to be more explicit like
`documentMode === 11 or documentMode === 10`.
let me know your thoughts?